routers/web/repo/actions/actions.go

```diff
@@ -85,7 +85,9 @@ func List(ctx *context.Context) {
 		}
 		allRunnerLabels := make(container.Set[string])
 		for _, r := range runners {
-			allRunnerLabels.AddMultiple(r.AgentLabels...)
+			if r.IsOnline() {
```

Comment:
I think we should change `FindRunnerOptions` to add a new option

Reply:
Done.